File: Client/client/src/app/models/PersonaNatural/PersonaNaturalSaveModel.ts
import { PersonaNaturalMedioComunicacionSaveModel } from "./PersonaNaturalMedioComunicacionSaveModel";

export class PersonaNaturalSaveModel {
    constructor(
        public Item: number = 0,
        public PersonaNaturalId: number = 0,
        public TipoDocumentoIdentidadId: number = 0,
        public NumDocumento: string = '',
        public Nombres: string = '',
        public ApellidoPaterno: string = '',
        public ApellidoMaterno: string = '',
        public FechaNacimiento: Date = new Date,
        public UbigeoId: number = 0,
        public Direccion: string = '',
        public Telefono: string = '',
        public Correo: string = '',
        public GeneroId: number = 0,
        public EstadoCivilId: number = 0,
        public FechaRegistro: Date = new Date,
        public CodUsuario: string = '',
        public EstadoRegistro: boolean = true,
        public Action: number = 0,
        public DetalleMedioComunicacion: PersonaNaturalMedioComunicacionSaveModel[] =[]

    ) { }
}

File: Desk/SpaceRT/Space.ServiceConsumer/Model/PersonaNaturalSaveModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Space.ServiceConsumer
{
    public class PersonaNaturalSaveModel
    {
        public Int32 PersonaNaturalId { get; set; }
        public Int32 TipoDocumentoIdentidadId { get; set; }
        public String NumDocumento { get; set; }
        public DateTime FechaRegistro { get; set; }
        public String CodUsuario { get; set; }
        public Int32 UbigeoId { get; set; }
        public String Nombres { get; set; }
        public String ApellidoPaterno { get; set; }
        public String ApellidoMaterno { get; set; }
        public DateTime FechaNacimiento { get; set; }
        public String Direccion { get; set; }
        public String Telefono { get; set; }
        public String Correo { get; set; }
        public Int32 SexoId { get; set; }
        public Int32 EstadoCivilId { get; set; }
    }
}
